PokeAlarm/Filters.py

```diff
@@ -47,7 +47,8 @@ def load_pokemon_section(settings):
         "min_sta": 0, "max_sta": 15,
         "quick_move": None, "charge_move": None, "moveset": None,
         "size": None,
-        "gender": None
+        "gender": None,
+        "min_cp": 0, "max_cp": 4760,
```

Comment:
hi, i added these code to my telegramalarm.py but telegram shows me an "?" instead of the cp:

# 'chat_id': If no default, required
	      'title': "<pkmn> CP: <cp>",
              'body': "bis: <24h_time> (<time_left>)."

whats wrong here ?